pyproject.toml

```diff
@@ -0,0 +1,61 @@
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry_dynamic_versioning.backend"
+
+[tool.poetry]
+name = "KaTrain"
+version = "0.0.0"  # Placeholder. The poetry-dynamic-versioning sets the version
+description = "Go/Baduk/Weiqi playing and teaching app with a variety of AIs"
+authors = ["Sander Land"]
+license = "MIT"
+homepage = "https://github.com/sanderland/katrain"
+readme = "README.md"
+packages = [{include = "katrain"}]
+include = [
+    {path = "katrain/KataGo/*"},
+    {path = "katrain/models/*"},
+    {path = "katrain/fonts/*"},
+    {path = "katrain/sounds/*"},
+    {path = "katrain/img/*"},
+    {path = "katrain/img/flags/*"},
+    {path = "katrain/i18n/*"}
```

Comment:
This logic from setup.py is more sophisticated, but the simpler configuration here seems to be equivalent.

```
package_data = {"": ["*.json", "*.kv", "*.wav"], "katrain": [], "tests": []}

def include_data_files(directory):
```

I ran the `python3 setup.py sdist` and `poetry build` with verbose mode and diffed their output to confirm that nothing extra is included.

Reply:
can we remove it entirely and just rely on the default? I can't remember why this was added in the first place 🤔

Reply:
Sure, I also verified that the same files get included.